app/assets/javascripts/miq_application.js

```diff
@@ -1632,6 +1632,27 @@ function miqHideSearchClearButton() {
   });
 }
 
+function miqSaveTreeDiv() {
```

Comment:
these 4 functions are unused, you can remove them (`miqSaveTreeDiv`, `miqLoadSavedTreeDiv`, `miqSetLeftCellTree` and `miqSetAngularPagingDivForButtons`)